src/CMakeLists.txt

```diff
@@ -1341,6 +1341,29 @@ if(HAVE_CLOCK_GETTIME)
    list( APPEND DEFINES HAVE_CLOCK_GETTIME )
 endif()
 
+set( AUDACITY_LIBRARIES
+# A sub-sequence of what is in libraries/CMakeLists.txt :
+   lib-string-utils-interface
+   lib-uuid-interface
+   lib-screen-geometry-interface
+   lib-project-rate-interface
+   lib-ffmpeg-support-interface
+   lib-theme-resources-interface
+   lib-track-interface
+   lib-transactions-interface
+)
+
+if ( ${_OPT}has_networking )
+   list( APPEND AUDACITY_LIBRARIES
+      lib-network-manager-interface
+
+      # This library depends on lib-network-manager
+      # If Sentry reporting is disabled, an INTERFACE library
+      # will be defined
+      lib-sentry-reporting
```

Comment:
Did you observe a build failure?  I omitted the `-interface` suffix.  That suffix is not necessary for builds, but adding it might sometimes break the build.

I could leave off `-interface` from everything in this list.  In fact I might have no good reason to add it.

I wrote the CMake stuff that generates the extra `-interface` alias targets because of some difficulties getting the modules, not the libraries, to build and link on Windows.

Reply:
> Did you observe a build failure? I omitted the `-interface` suffix. That suffix is not necessary for builds, but adding it might sometimes break the build.
> 
> I could leave off `-interface` from everything in this list. In fact I might have no good reason to add it.
> 
> I wrote the CMake stuff that generates the extra `-interface` alias targets because of some difficulties getting the modules, not the libraries, to build and link on Windows.

In other words are you implying that adding `-interface` may be necessary for the build when lib-sentry-reporting-interface is defined?  I think not.

Reply:
I do not imply anything about interfaces. I state the fact that lib-sentry-reporting always generates a target. If reporting is disabled - it generates a header-only library. This library is still required to build Audacity.

Reply:
So I don’t understand your objection to that line.  That target IS always present so will not cause a build error.

Reply:
It is in `if ( ${_OPT}has_networking )` block, which is wrong.

Reply:
You are correct.